File: snippets/python/src/webhook.py
import logging
from breez_sdk_liquid import BindingLiquidSdk


def register_webhook(sdk: BindingLiquidSdk):
    # ANCHOR: register-webook
    try:
        sdk.register_webhook("https://your-nds-service.com/notify?platform=ios&token=<PUSH_TOKEN>")
    except Exception as error:
        logging.error(error)
        raise
    # ANCHOR_END: register-webook

def unregister_webhook(sdk: BindingLiquidSdk):
    # ANCHOR: unregister-webook
    try:
        sdk.unregister_webhook()
    except Exception as error:
        logging.error(error)
        raise
    # ANCHOR_END: unregister-webook

File: snippets/csharp/Webhook.cs
using Breez.Sdk.Liquid;

public class ServiceStatusSnippets
{
    public void RegisterWebhook(BindingLiquidSdk sdk)
    {
        // ANCHOR: register-webook
        try
        {
            sdk.RegisterWebhook("https://your-nds-service.com/api/v1/notify?platform=ios&token=<PUSH_TOKEN>");
        }
        catch (Exception)
        {
            // Handle error
        }
        // ANCHOR_END: register-webook
    }
        
    public void UnregisterWebhook(BindingLiquidSdk sdk)
    {
        // ANCHOR: unregister-webook
        try
        {
            sdk.UnregisterWebhook();
        }
        catch (Exception)
        {
            // Handle error
        }
        // ANCHOR_END: unregister-webook
    }
}
